src/openforms/submissions/api/serializers.py

```diff
@@ -200,6 +210,13 @@ def to_representation(self, instance):
         check_submission_logic(instance, unsaved_data=self.context.get("unsaved_data"))
         return super().to_representation(instance)
 
+    def validate_object_reference(self, value: str) -> str:
+        re_match = re.search(r"^[a-zA-Z0-9\-]+$", value)
```

Comment:
This is almost the same regex as the `SlugField` one, so I think we should either:
- use `SlugField` at the model level, and just add `object_reference` in `Meta.fields` here (no need to define it explicitly)
- use `CharField` at the model level with a `RegexValidator`, and use a `RegexField` here in the serializer (I think DRF doesn't reuse model fields validators).

Reply:
Discussed this with Sergei as well on slack and decided to remove this for now since we do not know exactly the structure of the query parameter value. It will be a string so the `CharField` will do the needed validation.